plugins/video-intelligence-monetization/frontend/public/javascripts/countly.views.js

```diff
@@ -0,0 +1,182 @@
+      "sTitle": jQuery.i18n.map["common.date"]
+    }];
+
+    var eventColumns = countlyMonetization.getColumns();
+
+    for (var ekey in eventColumns) {
+      aoColumns.push({
+        "mData": ekey,
+        sType: "formatted-num",
+        "mRender": function(d) {
+          return countlyCommon.formatNumber(d);
+        },
+        "sTitle": jQuery.i18n.map[eventColumns[ekey].localize]
+      })
+    }
+
+    this.dtable = $('#dataTable').dataTable($.extend({}, $.fn.dataTable.defaults, {
+      "aaData": data,
+      "aoColumns": aoColumns
+    }));
+
+    $("#dataTable").stickyTableHeaders();
+  },
+  getData: function() {
+    return countlyMonetization.getMetricData();
+  },
+});
+
+window.monetizationIntegrationView = countlyView.extend({
+  beforeRender: function() {
+    var self = this;
+    if(!self.integrationTemplate){
+      return $.when($.get(countlyGlobal["path"] + '/video-intelligence-monetization/templates/integration.html', function(src) {
+        self.integrationTemplate = Handlebars.compile(src);
+      })).then(function(){});
+    }
+  },
+  renderCommon: function(isRefresh) {
+    $(this.el).html(this.integrationTemplate({}));
+    if (!window.monetization_iFrameResize) {
+      window.monetization_iFrameResize = function() {
+        var ifrm = iFrameResize({
+          interval: 5000,
+          log: false
+        }, '#vi-integration');
+        if (ifrm[0].iFrameResizer) {
+          window.monetization_iFrame = ifrm[0].iFrameResizer
+        }
+      }
+    }
+  },
+  refresh: function() {
+
+  },
+  destroy: function() {
+    if (window.monetization_iFrame) {
+      window.monetization_iFrame.close();
+      window.monetization_iFrame = null;
+    }
+    window.monetization_iFrameResize = undefined;
+  }
+});
+
+app.monetizationIntegrationView = new monetizationIntegrationView();
+app.monetizationMetricsView = new MonetizationMetricsView();
+
+app.route("/monetization/integration", 'monetization-integration', function() {
+  this.renderWhenReady(this.monetizationIntegrationView);
+});
+
+app.route("/monetization/metrics", 'monetization-metrics', function() {
+  this.renderWhenReady(this.monetizationMetricsView);
+});
+
+$(document).ready(function() {
+
+  var folder = '<a class="item" id="sidebar-monetization"><div class="logo ion-social-usd"></div><div class="text" data-localize="monetization.title"></div><span class="ion-chevron-right"></span></a>'
+  var items = [
+    '<a href="#/monetization/integration" class="item"><div class="text" data-localize="monetization.integration">Integration</div></a>',
+    '<a href="#/monetization/metrics" class="item"><div class="text" data-localize="monetization.metrics">Metrics</div></a>'
+  ]
+  var wrapper = '<div class="sidebar-submenu" id="monetization-submenu">' + items.join('') + '</div>'
+
+  if (!production) {
+    CountlyHelpers.loadJS("video-intelligence-monetization/javascripts/iframeResizer.js");
+  }
+
+  $('.sidebar-menu').append(folder + wrapper);
```

Comment:
This will add Monetization as last menu item. We usually try to leave Management as the last menu item, unless it was stated specifically that Monetization needs to be the last one